Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "attractorr"
-version = "0.4.2"
+version = "0.5.0"
```

```suggestion
version = "0.4.2"
```

Same here. I just bump the version before creating a release (where I decide if it is major or minor). This is to avoid conflicts when having multiple feature branches open.